### PR TITLE
Add support for FancyArrowPatch objects

### PR DESCRIPTION
**Pull request by eric-wieser**
See the black arrows added to `test_patch`:
image

### CODE REVIEW

**Comment by nschloe** [comment]
Just ping me when you think it's ready.

**Comment by eric-wieser** [comment]
Now works correctly for both my use case, and the test, just waiting on CI

File: test/test_patches_reference.tex [hunk 1]
```diff
@@ -198,6 +198,10 @@
 
 \draw[draw=none,fill=color0,fill opacity=0.4,rotate around={45:(axis cs:1,0.5)}] (axis cs:1,0.5) ellipse (0.5 and 0.25);
 \draw[draw=red,fill=red,opacity=0.4] (axis cs:0,1) circle (0.5);
+\draw[->,draw=black] (axis cs:0.25,0.25) -- (axis cs:0.5,0.25);
+\path [draw=black, -latex, draw=black]
```
Comment (eric-wieser) [inline test/test_patches_reference.tex]: `draw=` twice is a little sloppy, but probably harmless

**Comment by eric-wieser** [comment]
Would you mind looking at the CI failure? My browser can't scroll properly on that CircleCI page

**Comment by eric-wieser** [comment]
@nschloe: ready, ping

**Comment by nschloe** [comment]
LGTM, thanks!

**Pull request merged**